lib/web_ui/lib/src/engine/canvaskit/text.dart

```diff
@@ -729,12 +728,16 @@ class CkParagraph extends SkiaObject<SkParagraph> implements ui.Paragraph {
 
     for (int i = 0; i < skRects.length; i++) {
       final Float32List rect = skRects[i];
+      final Object? skTextDirection = getJsProperty(rect, 'direction');
+      final int skTextDirectionValue = skTextDirection != null
+          ? getJsProperty(skTextDirection, 'value')
+          : 0;
```

Comment:
Do we need to check for null? My gut feeling is that this is always available.

Reply:
No. Done